libs/libapi/pyproject.toml

```diff
@@ -14,6 +14,7 @@ orjson = "^3.9.15"
 pyjwt = { extras = ["crypto"], version = "^2.6.0" }
 starlette = "^0.37.1"
 starlette-prometheus = "^0.9.0"
+huggingface-hub = {extras = ["hf-transfer"], version = "^0.21.4"}
```

Comment:
maybe define it in libcommon instead (it's already a dependency of libcommon, only missing the `extras = ["hf-transfer"]` part)